ios/autotools-ios-helper.sh

```diff
@@ -63,8 +63,8 @@ build_iphoneos() {
     PLATFORM="iphoneos"
     ARCH_OPTS="--enable-armasm --enable-sp-asm"
     ARCH_FLAGS="-arch arm64"
-    HOST_FLAGS="${ARCH_FLAGS} -mfpu=auto -miphoneos-version-min=${MIN_IOS_VERSION} -isysroot $(xcrun --sdk ${SDK} --show-sdk-path)"
-    CHOST="aarch64-apple-darwin"
+    HOST_FLAGS="${ARCH_FLAGS} -miphoneos-version-min=${MIN_IOS_VERSION} -isysroot $(xcrun --sdk ${SDK} --show-sdk-path)"
```

Comment:
`-mfpu=auto` was working before with previous WolfSSL releases before. What are the reasons that we have to remove it here?

Reply:
Removed this here, because this should be the default option. Overall it makes more sense to allow the compiler to determine what the correct fpu setting is via the `iphoneos-version-min`